.github/CODEOWNERS

```diff
@@ -1,24 +1,28 @@
 # Learn how to add code owners here:
 # https://help.github.com/en/articles/about-code-owners
 
-*          @timneutkens @ijjk @shuding @huozhi @feedthejim
-/.github/  @vercel/next-js
-/docs/     @timneutkens @ijjk @shuding @huozhi @feedthejim @styfle @padmaia @leerob @balazsorban44 @jankaifer
-/errors/   @timneutkens @ijjk @shuding @huozhi @feedthejim @styfle @padmaia @leerob @balazsorban44 @jankaifer
-/examples/ @timneutkens @ijjk @shuding @feedthejim @leerob @steven-tey @balazsorban44 @jankaifer
+*                                             @timneutkens @ijjk @shuding @huozhi @feedthejim
+/.github/                                     @vercel/next-js
+/docs/                                        @vercel/next-js @leerob
+/errors/                                      @vercel/next-js @leerob
+/examples/                                    @vercel/next-js @leerob @steven-tey
+/scripts/                                     @vercel/next-js
+/.*                                           @vercel/next-js @leerob
```

Comment:
What other `.` folders should the whole team be requested on?

Reply:
It's dotfiles, not the folders. :+1:

Reply:
Is this matching only folders or does it match the dot files too?

Reply:
Had to double-check:

https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

https://git-scm.com/docs/gitignore#_pattern_format

> An asterisk "*" matches anything except a slash.

So as I understand, this only matches files (which was my intention in the PR)